book/src/ui-authentication.md

```diff
@@ -1,31 +1,5 @@
-# Authentication
-
-To enhance the security of your account, we offer the option to set a session password. This allows the user to avoid re-entering the api-token when performing critical mutating operations on the validator. Instead a user can simply enter their session password. In the absence of a session password, Siren will revert to the api-token specified in your configuration settings as the default security measure.
-
-> This does not protect your validators from unauthorized device access.
-
-![authentication](imgs/ui-session-auth.png)
-
-Session passwords must contain at least:
-
-- 12 characters
-- 1 lowercase letter
-- 1 uppercase letter
-- 1 number
-- 1 special character
-
 ## Protected Actions
```

Comment:
If this is the first line of the file, it needs to be a top-level heading with only 1 `#`: https://github.com/DavidAnson/markdownlint/blob/main/doc/md041.md
```suggestion
# Protected Actions
```